src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java

```diff
@@ -14,8 +15,13 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
     {
         TFM_Util.adminAction(sender.getName(), "Opping " + sender.getName(), false);
         sender.setOp(true);
-        sender.sendMessage(TFM_Command.YOU_ARE_OP);
-
+            sender.sendMessage(TotalFreedomMod.YOU_ARE_OP);
```

Comment:
This should have a if not imposer wrapped around it, then have an else for if they are to send them the message that they cannot get OP